ui/app/templates/components/auth-info.hbs

```diff
@@ -12,6 +12,16 @@
                 on {{date-format auth.tokenExpirationDate 'MMMM Do YYYY, h:mm:ss a'}}" />
           </li>
         {{/if}}
+        <li class="action">
+          <button type="button" class="link" onclick={{action "restartGuide"}}>
+            Restart guide
+          </button>
+        </li>
+        <li class="action">
+          <CopyButton @clipboardText={{auth.currentToken}} class="link" @buttonType="button" @success={{action (set-flash-message 'Token copied!')}}>
+            Copy token
+          </CopyButton>
+        </li>
```

Comment:
Gathering the "destructive actions" at the bottom of the list, so moving these up